### DISCUSSION TITLE
Idea: Endless Ammo mutator

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1184] [Ideas and Feature Requests]
In 2.0.0, there is an ammo cap. Why not add an endless ammo mutator?

**Comment by byronsharman** [comment]
Since all the gameplay is based around having limited ammo, wouldn't endless ammo cause some weapons to be more spammed all the time? It might make certain combinations more powerful.

**Comment by ghost** [comment]
But then don't choose it or don't vote it. Some people like to use endless ammo.

**Comment by ghost** [comment]
Also, SMG and zapper have a lot of recoil. Spamming it would be hard.

**Comment by qreeves** [comment]
Moving to `world` as the issues on `base` aren't the place for random discussion.

**Comment by ghost** [comment]
> Since all the gameplay is based around having limited ammo, wouldn't endless ammo cause some weapons to be more spammed all the time? It might make certain combinations more powerful.

It was the case in the first, people used more the flammnoob but combinations were equalized and balanced.
